dev-resources/logback.xml

```diff
@@ -5,11 +5,11 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="error"/>
+    <logger name="puppetlabs" level="trace"/>
```

Comment:
Should we have a logback-dev and a logback-test? I've seen other projects do that.